keyboards/launchpad/rev1/rev1.h

```diff
@@ -0,0 +1,24 @@
+#ifndef REV1_H
+#define REV1_H
+
+#include "../launchpad.h"
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
+
+//void promicro_bootloader_jmp(bool program);
+
+#define KEYMAP( \
```

Comment:
We're deprecating KEYMAP in favor of LAYOUT. Would you change all instances of KEYMAP to LAYOUT?